opm/simulators/utils/BlackoilPhases.hpp

```diff
@@ -60,6 +60,7 @@ namespace Opm
         bool has_foam{};
         bool has_brine{};
         bool has_zFraction{};
+        bool has_co2_or_h2store{};
 
```

Comment:
Adding has_co2_or_h2store is ok I think. Maybe we should consider renaming the BlackoilPhases.hpp at some point.